app/src/modules/win.js

```diff
@@ -92,9 +92,13 @@ const win = {
     const handleRedirect = (e, url) => {
       try {
         const urlData = new URL(url);
-        const isAllowedUrl = WHITE_LISTED_DOMAIN.includes(urlData.origin);
 
-        if (!isAllowedUrl) return e.preventDefault();
+        const isUrlAllowed = WHITE_LISTED_URLS.some(
```

Comment:
Couldnt we have a function for this `isUrlAllowed(url)` which we import here instead of `WHITE_LISTED_URLS`? Bc the WHITE_LISTED_URLS  is a pretty complex object but the important part is the implementation which would be shown easily in a function.